plugins/op-z/export.js

```diff
@@ -0,0 +1,130 @@
+const {
+  CoarseChannel,
+  FineChannel,
+  NullChannel,
+  SwitchingChannel
+} = require(`../../lib/model.js`);
+
+module.exports.name = `OP-Z`;
+module.exports.version = `0.1.0`;
+
+const MAX_KNOBS = 8;
+const MAX_OPZ_FIXTURES = 16;
+
+/**
+ * @param {array.<Fixture>} fixtures An array of Fixture objects.
+ * @param {object} options Global options, including:
+ * @param {string} options.baseDir Absolute path to OFL's root directory.
+ * @param {Date|null} options.date The current time.
+ * @returns {Promise.<array.<object>, Error>} The generated files.
+*/
+module.exports.export = function exportOpZ(fixtures, options) {
+  const exportJson = {
+    profiles: [],
+    config: []
```

Comment:
Maybe move `config` above `profiles`? This will make it easier to edit the JSON for just choosing another fixture.

Reply:
I'd like to keep the order the same as the default `dmx.json` provided by the OP-Z. Also, it doesn't really matter since the plugin will be most useful for just exporting a small number of fixtures and then manually editing the resulting file.